src/07-projects/milestones.apib

```diff
@@ -101,6 +101,12 @@ Create a new milestone.
         + due_on: `2018-01-01` (string, required)
         + name: `Initial setup` (string, required)
         + responsible_user_id: `e1240972-6cfc-4549-b49c-edda7568cc48` (string, required)
+        + `invoicing_method`: `time_and_materials` (enum[string])
```

Comment:
Q: Could we name this something like `pricing_model`? We might have some mixed concepts between an invoicing method for invoicing endpoints which could be PayPal, credit card, bank transfer and invoicing method in this context.

If I search for time-and-materials vs. fixed price they often refer to it as the project pricing model.

Reply:
> Q: Could we name this something like `pricing_model`? We might have some mixed concepts between an invoicing method for invoicing endpoints which could be PayPal, credit card, bank transfer (I don't know) and invoicing method in this context. If this is indeed the correct term ignore the comment. But if we could avoid potential confusion it might be worth double checking :)
> 
> If I search for time-and-materials vs. fixed price they often refer to it as the project pricing model.

My 2 cents:
We already provide this information on `milestones.list` and `milestones.info` endpoint, so it wouldn't make sense to have different (field) names for the same purpose/fields. 
The term that is used throughout the UI in project milestones is actually `billing method`.

Reply:
I do prefer billing method over invoicing method indeed.